### PR TITLE
Jean/cow 345 create minimal airdrop hook UI

### PR DESCRIPTION
**Pull request by jean-neiverth**
Added an Airdrop Hook option on Sepolia network

image

### CODE REVIEW

File: apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+        </Row>
+        {/* <Row>
+          <Input
+            name="gasLimit"
+            type="number"
+            placeholder='Gas Limit'
+            value={hook.gasLimit}
+            onChange={(e) => setHook((hook) => ({ ...hook, gasLimit: e.target.value }))}
+          />
+        </Row> */}
```
Comment (yvesfracari) [inline apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx]: 👀

File: apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+    justify-content: space-between;
+    cursor: pointer;
+    border-radius: 16px;
+    background-color: #18193B;
+    padding:1rem;
+    width:100%;
+
+    &:hover {
+    background-color: #65D9FF;
+    color:#000000;
```
Comment (yvesfracari) [inline apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx]: How do they handle the colors of the other components? Probably there is some kind of variable to make it easier to update, right? Can you check this?

File: apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    }
+
+`
+
+const airdrops = [
```
Comment (yvesfracari) [inline apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx]:
```suggestion
const AIRDROP_OPTIONS = [
```

Comment (yvesfracari) [reply apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx]: maybe we should separate this constants in another file?

File: apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+    },
+]
+
+export function AirdropHookApp() {
+  const hookDappContext = useContext(HookDappContext)
+  const [hook, setHook] = useState<CowHook>({
+    target: 'test',
+    callData: 'test',
+    gasLimit: 'test',
+  })
```
Comment (yvesfracari) [inline apps/cowswap-frontend/src/modules/hooksStore/dapps/AirdropHookApp/index.tsx:208]: #would mark this temp code with a TODO tag and the linear issue that should handle this.

**Review by yvesfracari — APPROVED** [review]

**Comment by jean-neiverth** [comment]
Corrected review points

**Pull request merged**